provision/provision-site.sh

```diff
@@ -84,6 +84,10 @@ function get_primary_host() {
 function vvv_provision_site_nginx_config() {
   local SITE_NAME=$1
   local SITE_NGINX_FILE=$2
+  if [ ! -f "$SITE_NGINX_FILE" ]; then
+    vvv_warn " * VVV didn't found any Nginx config file, it will use a fallback config file"
+    cp -f "/srv/provision/core/nginx/config/sites/fallback.conf" "$SITE_NGINX_FILE"
+  fi
```

Comment:
this scenario isn't possible because the code that calls `vvv_provision_site_nginx_config` already performs this check, and if nothing is found it moves on to the next thing. If nothing is found then this function is never called.

`vvv_provision_site_nginx` would need changes to support this too

Reply:
I would imagine instead this part of the `vvv_provision_site_nginx` function would need changing:

```sh
    if [[ -z $NGINX_CONFIGS ]] ; then
      vvv_error " ! Error: No nginx config was found, VVV will not know how to serve this site"
      exit 1
```

So that it called `vvv_provision_site_nginx_config` passing the fallback config as the parameter